### ISSUE TITLE
pachyonychia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4623]
see https://www.ncbi.nlm.nih.gov/pubmed/20004881

**Comment by pnrobinson** [comment]
Figure 1 https://www.ncbi.nlm.nih.gov/pubmed/12673665

**Comment by pnrobinson** [comment]
actually, this word is used to describe a disease that is characterized by this kind of nail defect
https://hpo.jax.org/app/browse/term/HP:0001805
I think we should probably switch the main label to Onychogryposis

**Comment by pnrobinson** [comment]
Switched primary label.
